File: Data Structures & Algorithms/max-consecutive-ones/submission-0.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int num =0; int prev=0;
        for(int n : nums){
            if(n==1) { 
                num++;
                prev=Math.max(num,prev);
            }
            else if(n==0) {
                num=0;
            }
        }
        return prev;
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.cpp
class Solution {
public:
    int longestConsecutive(vector<int>& nums) {
        if (nums.empty()) return 0;
        sort(nums.begin(), nums.end());
        int lcs=1, cur=1;
        for (int i=1;i<nums.size();i++) {
            if (nums[i]==nums[i-1]) {
                continue;
            }
            if (nums[i]==nums[i-1]+1) {
                cur++;
            } 
            else {
                lcs=max(lcs,cur);
                cur=1;
            }
        }
        return max(lcs, cur);
    }
};
